css/_settings.scss

```diff
@@ -120,3 +131,9 @@ $header-size: rem(80);
 $out-space: rem(110);
 $ui-space: rem(15);
 $margin-size: rem(40);
+
+
+// Bootstrap overrides
+
+/* remove rounding from cards, buttons and inputs */
+.card, .btn, .form-control, .dropdown-menu { border-radius: 0; }
```

Comment:
You don't need to override the styles, you can turn off Boostrap's `border-radius`.

You have two options:
- Either you remove all of them
- Or you remove them only for cards, buttons, inputs and menus

If you go with the first solution, set these variables to `0`: `$border-radius`, `$border-radius-lg` and `$border-radius-sm`.

If you go with the second one, there are way more variables to set. You can find the full list in `node_modules/bootstrap/scss/_variables.scss`.

I think you should go with the first solution, and if we need the `border-radius` back, we can selectively enable them.